go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/editorconfig-checker/editorconfig-checker/v3
 
 go 1.21
+toolchain go1.22.5
```

Comment:
This is probably the root of the error.

Reply:
which in turn is caused by go-snaps switching to go 1.22. But since we are end-consumers and I would not assume anybody consuming our package as a library, we could also easily always use the latest release of go, so using go1.23 from now on.